Chapter09/recipe-06/cxx-example/CMakeLists.txt

```diff
@@ -1,14 +1,9 @@
 # define minimum cmake version
-cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
 
 # project name and supported language
 project(recipe-06 CXX)
 
```

Comment:
This misses the C++11 settings. I see they were removed, why?

Reply:
I don't think they were removed for a good reason. I will re-add.